compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -6,7 +6,7 @@ pub use self::IntVarValue::*;
 pub use self::Variance::*;
 
 use crate::hir::exports::ExportMap;
-use crate::hir::place::Place as HirPlace;
+use crate::hir::place::{Place as HirPlace, PlaceBase, ProjectionKind};
```

Comment:
can you add PlaceBase as HirPlaceBase and similarly for ProjectionKind. and ideally each one of the imports on their own lines.